engine/src/test/java/com/datatorrent/stram/plan/logical/module/TestModuleExpansion.java

```diff
@@ -184,7 +190,7 @@ public void setPortMemory(int portMemory)
     }
   }
 
-  static class Level2ModuleA implements Module
+  public static class Level2ModuleA extends BaseOperator implements Module
```

Comment:
These changes are required for test cases to pass. without public newInstance invocation fails in addOperator.  There are similar test operator which are defined as public for example TestGeneratorInputOperator

I am removing extends BaseOperator from modules, they were leftovers from previous implementation where module was extending operator.